src/fontra/views/editor/editor.js

```diff
@@ -183,9 +183,29 @@ export class EditorController {
     this.initContextMenuItems();
     this.initShortCuts();
     this.initMiniConsole();
-    this.initPlugins().then(() => {
-      this.restoreOpenTabs("left");
-      this.restoreOpenTabs("right");
+
+    let waitPluginsToRestoreTabs = [];
```

Comment:
I'll need a better variable name

Reply:
This holds sidebar names

Reply:
deferredPanels maybe?

Could it be a const?

Reply:
Yes, it can be a const.

Reply:
Ah, I know see I misunderstood what waitPluginsToRestoreTabs actually stored. Maybe `deferRestoreOpenTabs`.